dra-evolution/testdata/pools-two-nodes-dgxa100.yaml

```diff
@@ -0,0 +1,7695 @@
+- apiVersion: resource.k8s.io/v1alpha3
+  kind: ResourcePool
+  metadata:
+    creationTimestamp: null
+    name: nvidia-00-dgxa100
+  spec:
+    attributes:
+    - name: brand
+      string: Nvidia
+    - name: architecture
+      string: Ampere
+    - name: cuda-compute-capability
+      version: 8.0.0
+    - name: driver-version
+      version: 550.54.15
+    - name: cuda-driver-version
+      version: 12.4.0
+    - name: product-name
+      string: Mock NVIDIA A100-SXM4-40GB
+    devices:
+    - attributes:
+      - name: uuid
+        string: GPU-cdd0dd02-b5dd-4ae9-9c5a-c76b19086eb4
+      - bool: true
+        name: mig-capable
+      - bool: false
+        name: mig-enabled
```

Comment:
NOTE: we should establish core guidelines - are attributes expected to be mutable or immutable?  Do we encourage drivers to embed status here or are attributes only *intrinsic* things about the device?  This one feels like status.

Reply:
This should it be an attribute. MIG capable yes, enabled no. Thisbia leftover from the structure I sided to represent all of this in classic DRA. It is very compact, but also not generalizable.